networkit/cpp/community/CoverHubDominance.cpp

```diff
@@ -56,7 +56,7 @@ void NetworKit::CoverHubDominance::run() {
 
             double dominance = 1;
             if (clusterSizes[i] > 1) {
-                dominance = maxInternalDeg[i] * 1.0 / (clusterSizes[i] - 1);
+                dominance = maxInternalDeg[i] * 1.0 / (clusterSizes[i] - 1.0);
```

Comment:
static_cast around the denominator (+ revert to 1) + static_cast around `maxInternalDeg[i]` (uint64 -> double).